src/sentry/integrations/metric_alerts.py

```diff
@@ -217,9 +219,34 @@ def incident_attachment_info(
     if notification_uuid:
         title_link_params["notification_uuid"] = notification_uuid
 
-    if features.has("organizations:workflow-engine-trigger-actions", organization) and features.has(
-        "organizations:workflow-engine-ui-links", organization
-    ):
+    if features.has("organizations:workflow-engine-trigger-actions", organization):
+        try:
+            alert_rule_detector = AlertRuleDetector.objects.get(
+                detector_id=alert_context.action_identifier_id
+            )
```

Comment:
nit: since you're just grabbing the `alert_rule_id` you can do:

`AlertRuleDetector.values_list("alert_rule_id", flat=True).get(detector_id=alert_context.action_identifier_id)`

and also just double checking here that the `action_identifier_id` is a detector id?

Reply:
and same deal for the `IncidentGroupOpenPeriod` to just grab the incident id